changelog.txt

```diff
@@ -1128,10 +1132,10 @@ Renamed method annotations used by the Nullness Checker:
 Removed annotations used by the Nullness Checker:
   @AssertParametersNonNull
 Renamed type annotations used by the Initialization Checker:
-  @NonRaw => @Initialized
```

Comment:
The changes in the file should be reverted, except for the added entry at the top.